src/components/expanded-state/liquidity-pool/LiquidityPoolExpandedStateHeader.js

```diff
@@ -45,7 +46,15 @@ function LiquidityPoolExpandedStateHeader({ assets, pricePerShare }) {
   const shadows = useMemo(() => {
     return map(assets, (asset, index) => {
       const coinIconShadow = [
-        [0, 4, 12, asset.shadowColor || colors[index], 0.3],
+        [
+          0,
+          4,
+          12,
+          asset.symbol === 'ETH'
```

Comment:
can we do asset.address === 'eth' so we're relying less on symbols (only the real ETH will have this address)